### DISCUSSION TITLE
Searching and listing repositories by code search

### DISCUSSION THREAD

**Discussion by jurgen-kluft** [community/community#33406] [Other Feature Feedback, Questions, & Ideas] [answered]
When using advanced search to find repositories it seems not possible to list repositories by code search.
Currently you can search code but then you get a list of files that are part of a repository. I would like to just see the list of repositories that got hit by the code search.

**Comment by look** [comment, marked as answer]
Thanks for this feedback. We've heard similar requests from other people. Right now we're focused on releasing the new Code Search experience, so I can't promise any timeline but we will consider this.
